include/ppx/geometry.h

```diff
@@ -146,8 +153,14 @@ class Geometry
     {
     public:
         Buffer() {}
-        Buffer(BufferType type, uint32_t elementSize)
-            : mType(type), mElementSize(elementSize) {}
+        Buffer(BufferType type, uint32_t elementSize, uint32_t knownElementCount = 0)
```

Comment:
Can this be `maxElementCount` to keep the names consistent across the classes?

Reply:
Done